README.md

```diff
@@ -8,11 +8,12 @@ This repository allows you to quickly install Apache Solr for Drupal 9 into a [D
 
 1. `ddev get drud/ddev-drupal9-solr && ddev restart`
 1. You may need to install the relevant Drupal requirements: `ddev composer require drush/drush:* drupal/search_api_solr`
```

Comment:
We should also include the `drupal/search_api` module here, otherwise the URL in the next step is a 404.

Reply:
Oh, I see. It's a dependency. However the steps don't explicitly state to 'enable the module'.
IIRC, there was a recipe that required config before enabling the module (?) so I did not enable it yet.

Reply:
Thanks. I just committed the README change suggested by @rfay about that.